testonly/hammer/hammer.go

```diff
@@ -127,7 +127,13 @@ type MapConfig struct {
 	EPBias        MapBias
 	Operations    uint64
 	EmitInterval  time.Duration
-	IgnoreErrors  bool
+	IgnoreErrors, CheckSignatures bool
```

Comment:
Could we have a TODO to remove this when all map implementations have signatures.

Reply:
Done.  Also, added the equivalent TODO beside the flag definition.